### PR TITLE
Introduces Swiftlint for .swift files

### PR DESCRIPTION
**Pull request by surpher**
- excludes tests (written in ObjC)
- executes swiftlint on TravisCI
- fixes the deprecated method (started in iOS9)

### CODE REVIEW

File: PactConsumerSwiftTests/MatcherSpec.swift [hunk 1]
```diff
@@ -66,7 +66,7 @@ class MatcherSpec: QuickSpec {
       }
 
       it("allows min to be specified") {
-        subject = Matcher.eachLike(arrayItem, min: 4)
+        subject = Matcher.eachLike(arrayItem, minValue: 4)
```
Comment (andrewspinks) [inline PactConsumerSwiftTests/MatcherSpec.swift]: Can we leave this as min and put in an exclusion? Changing the matcher's api will break any users who are relying on this matcher in their tests.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
-- scripts/import_keychain.sh
-- security find-identity -v -p codesigning
-- gem install pact-mock_service -v 0.12.0
-- bundle install
-- carthage checkout
+  - scripts/import_keychain.sh
+  - security find-identity -v -p codesigning
+  - gem install pact-mock_service -v 0.12.0
+  - bundle install
+  - scripts/install_swiftlint.sh
```
Comment (andrewspinks) [inline .travis.yml]: Could swiftlint be installed via homebrew? (brew should be installed on travisci). Might be simpler to maintain....

**Review by andrewspinks — CHANGES_REQUESTED** [review]
I think we should leave the 'min' attribute name in the eachLike matcher otherwise we will break clients.

File: PactConsumerSwiftTests/MatcherSpec.swift (hunk 1 above)
Comment (surpher) [reply PactConsumerSwiftTests/MatcherSpec.swift]: Sure. Min 4 character var names rule will exclude the `min` and `max` var names.

File: .travis.yml (hunk 2 above)
Comment (surpher) [reply .travis.yml]: Will take longer to run the pipe. But will use brew update/upgrade in the script.

**Pull request merged**